_posts/2020-08-01-newsletter.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Origami Newsletter, July 2020
+description: This issue features a tutorial to create a new Origami component, automated "readme" documentation review, and new "indicator" labels.
+author: Lee Moody
+tags:
+  - Newsletter
+---
+
+<abbr title="Too long; didn't read">
+	<strong>
+	TL;DR:
+	</strong>
+</abbr> {{page.description}}
+
+## Top Things
+
+These are some of the bigger things we've done over the last month.
+
+### Create A New Origami Component
+
+This month we released a new tutorial, [Create A New Origami Component](https://origami.ft.com/docs/tutorials/create-a-new-component-part-1/). The tutorial starts from an empty folder and concludes by publishing a new component to the [Origami registry](https://registry.origami.ft.com/components).
+
+Our aim was to encourage more contributions from outside the core Origami team by pulling together documentation from various sources into a complete step-by-step guide. We hope the tutorial will be useful for teams who want to create new Origami components, such as when the former Comments team built [o-comments](https://registry.origami.ft.com/components/o-comments@7.6.5); and for new or guest members of the Origami team, such as when Olga Averjanova created the [o-meter](https://registry.origami.ft.com/components/o-meter@2.0.4?brand=internal) component during their bootcamp with us.
+
+The tutorial is comprehensive and covers topics which may not be relevant to all components. In the future we would like to reduce the length of the tutorial by splitting out topics into separate tutorials.
+
+If you are interested in creating a component please take a look. We are available in the [#origami-support](https://app.slack.com/client/T025C95MN/C02FU5ARJ) Slack channel to help if you get stuck, and as always welcome feedback or suggestions on all things Origami in the [#origami-chat](https://app.slack.com/client/T025C95MN/CSW6B2VAN) Slack channel 😀
+
+### Automated Readme Review
+
+Another change we've made to help Origami contributors and Origami users is to verify README documentation of a component. The latest version of [Origami Build Tools](https://github.com/Financial-Times/origami-build-tools) checks for:
+- A title which matches the component name.
+- A short description of the component.
+- A table of contents with matching heading levels.
+- A licence.
+
+Any issues are also flagged directly within Github.
+
+### Indicator Labels
+
+Working with the Content Innovation team we added new indicator labels such as the "live" and "new" story labels to the `o-labels` component. Previously the labels were implemented separately in multiple projects such as for teasers and the live blogs page. Maintaining two implementations may cause future design inconsistency or require duplicate effort to fix bugs. Now all projects can include `o-labels` instead of reinvent the indicator labels.
```

Comment:
```suggestion
Working with the Content Innovation team we added new indicator labels such as the "live" and "new" story labels to the `o-labels` component. Previously the labels were implemented separately in multiple projects such as for teasers and the live blogs page. Maintaining two implementations may cause future design inconsistency or require duplicate effort to fix bugs. Now all projects can include `o-labels` instead of reinventing the indicator labels.
```